packages/angular/ng-package.json

```diff
@@ -1,5 +1,6 @@
 {
   "$schema": "./node_modules/ng-packagr/ng-package.schema.json",
+  "dest": "../../bazel-out/darwin-fastbuild/bin/packages/angular/ng-pkg",
```

Comment:
This is a hack and needs to be properly fixed. Still discussing with @alexeagle on how best to do this.